_docs/_latest/concepts/architecture.md

```diff
@@ -22,21 +26,23 @@ security risks.
  * Notifier keeps you up to date about Forseti findings and actions.
  * Explain helps you understand, test, and develop Cloud Identity and Access
    Management (Cloud IAM) policies.
+   
+---
 
-The image below shows how data flows through Forseti:
+**The image below shows how data flows through Forseti**
 
 {% responsive_image path: images/docs/concepts/forseti-architecture.png alt: "forseti architecture" %}
 
- 1. Inventory collects information about your GCP resources and G Suite Groups.
- 2. Inventory stores information in Cloud SQL for your review and use by other
+ 1. Inventory collects information about your GCP resources and G Suite Groups and Users.
+ 1. Inventory stores information in Cloud SQL for your review and use by other
```

Comment:
I had numbered these manually to make sure that they followed the numbering on the chart. Do you think it's "safe" to allow auto-numbering? (e.g., if the chart changes, it could be harder to verify if the numbers are correct).

Reply:
Ah you're correct, fixed.